db/migrate/20221102134032_add_bios_to_host_facets.rb

```diff
@@ -0,0 +1,13 @@
+class AddBiosToHostFacets < ActiveRecord::Migration[6.1]
+  def up
+    add_column :host_facets_reported_data_facets, :bios_vendor, :string, :limit => 255
```

Comment:
Not a blocker for this PR, but we should at some point consider some JSON field instead to store all values. It'll make future additions easier. And looking at the various UI changes I get the feeling we want to utilize this pattern more.

In case you want to read up on it, we already have one instance in our codebase:
https://github.com/theforeman/foreman/blob/3146fd8647194bd5f821e9951c7ff94521d509f6/app/models/smart_proxy_feature.rb#L7